invenio_opendata/testsuite/data/cms/cms-learning-resources.xml

```diff
@@ -228,4 +228,47 @@
       <subfield code="b">Education</subfield>
     </datafield>
   </record>
+  <record>
+    <controlfield tag="001">61</controlfield>
+    <datafield tag="041" ind1=" " ind2=" ">
+      <subfield code="a">English</subfield>
+    </datafield>
+    <datafield tag="100" ind1=" " ind2=" ">
+      <subfield code="a">Lehti, Sami</subfield>
+      <subfield code="u">University of Helsinki</subfield>
+    </datafield>
+    <datafield tag="245" ind1=" " ind2=" ">
+      <subfield code="a">Computing Methods in High Energy Physics</subfield>
+    </datafield>
+    <datafield tag="260" ind1=" " ind2=" ">
+      <subfield code="b">University of Helsinki</subfield>
+      <subfield code="c">2014</subfield>
+    </datafield>
+    <datafield tag="520" ind1=" " ind2=" ">
+      <subfield code="a">This introductory course by S. Lehti is aimed for students with no prior knowledge of computing methods in high-energy physics. Prior knowledge of some programming language is needed. The course goes through basics of programming languages in high energy physics (FORTRAN, C++, python), software for analysis and visualisation (ROOT), some software for calculating cross sections and branching ratios, simulation of collisions (event generation), basics of CMS software (CMSSW) and an introduction to grid computing. This course is developed in the University of Helsinki for students interested in experimental particle physics.</subfield>
+    </datafield>
```

Comment:
Good point, in the title, I copied the original from the homepage and then I tried to follow @RaoOfPhysics wish for the rest.